cityofaustin/joplin
# Issue 4776: implement searchIndex for Elasticsearch

https://github.com/cityofaustin/techstack/issues/4776

# Description

- [ ] Cleaned up our api to help with searchIndex building. Now we have inheritance. All pageType nodes inherit from JanisBasePageNode. This is good because we no longer have to duplicate resolvers for fields that exist on every content_type.
- [ ] Added a "search_summary" field and resolver. Each content_type's model will be responsible for providing the specific field they want to use as a search summary.

Janis PR: https://github.com/cityofaustin/janis/pull/853

# Types of changes



# How can this be tested?
https://janis-v3-4776-elastic.netlify.app/en/search/#page=1
This search should work the same as it did before. 
- [ ] Make sure all the page results we'd expect to see show up.
- [ ] And that those links work.
- [ ] And that the right summary/data below the link shows up for correctly for each content_type.

# Checklist:
